package.json

```diff
@@ -93,55 +94,55 @@
       "types": "./esm/index.d.ts",
       "module": "./esm/index.js",
       "import": "./esm/index.js",
-      "default": "./lib/index.js"
+      "default": "./esm/index.js"
```

Comment:
I don't feel confortable about this change. We will pause this till July.

Left more context here
https://github.com/coinbase/onchainkit/pull/616